_docs_use-cases/use-case-consumption-get-attributes.md

```diff
@@ -0,0 +1,68 @@
+---
+# Start automatic generation
+permalink: use-case-consumption-get-attributes
+redirect_from:
+  - /use-case-consumption-query-attributes
+published: true
+title: "Get Attributes"
+type: use-case
+toc: true
+sidebar:
+  - title: "Integrate enmeshed"
+    nav: "docs_integrate"
+properties:
+  - id: RA2
+  - component: Runtime
+  - layer: Consumption
+  - facade: AttributesFacade
+  - function: getAttributes
+  - description:
+  - feature category: Normalized Attributes
+  - tech category: Attributes
+  - status: DONE
+  - documentation status: DONE
+  - comments:
+  - actor: Identity
+  - trigger: REST API
+  - precondition:
+  - result:
+  - priority: n/a
+  - complexity: n/a
+  - size: n/a
+  - created_at:
+  - changed_at:
+  - api_route_regex: GET /api/v2/Attributes
+  - published: default
+  - link: use-case-consumption-get-attributes
+require:
+required_by:
+api_route_regex: ^GET /api/v2/Attributes$
+# End automatic generation
+---
+
+{{properties.description}}
+
+{% include properties_list.html %}
+
+This use case is intended to query [LocalAttributes]({% link _docs_integrate/data-model-overview.md %}#localattribute). The LocalAttributes can be specified using a complex query.
+
+## Parameters
+
+- `query` allows to specify the conditions for the returned LocalAttributes. In detail, the following keys may be used:
+  - `createdAt` describes the time when the LocalAttribute was created.
+  - `parentId` can be used to find the child Attributes of a [complex IdentityAttribute]({% link _docs_integrate/attribute-introduction.md %}#complex-identityattributes).
```

Comment:
the page https://[doc-demo.is.enmeshed.eu/docs/json](https://doc-demo.is.enmeshed.eu/docs/json) is regularly updated? Who does it and when and how? Because
"description": "Query Attributes", should be "description": "Get Attributes",
and the parameter of `parentId` is totally missing. @Milena-Czierlinski @jkoenig134

Reply:
it isn't. But I can do it now :)

Reply:
it's now on latest